docs/parameters/validation.md

```diff
@@ -114,6 +114,17 @@ params.validationFailUnrecognisedParams = true
         --8<-- "examples/validationFailUnrecognisedParams/pipeline/main.nf"
         ```
 
+## Ignoring unrecognized parameters
+
+Sometimes, a parameter that you want to set may not be described in the pipeline schema for a good reason.
+Maybe it's something you're using in your Nextflow configuration setup for your compute environment,
+or it's a complex parameter that cannot be handled in the schema.
+
+In these cases, to avoid getting warnings when that unrecognised parameter is set,
+you can use `--validationSchemaIgnoreParams` / `params.validationSchemaIgnoreParams`.
```

Comment:
```suggestion
you can use `--validationSchemaIgnoreParams` / `params.validationSchemaIgnoreParams = true`.
```
?

Reply:
This parameter should be a list of the parameters to ignore, it's explained in the next line